test/correctness/float_helpers.h

```diff
@@ -0,0 +1,40 @@
+#ifndef FLOAT_HELPERS_H
+#define FLOAT_HELPERS_H
+#include <cinttypes>
+uint32_t bits_from_float(float v) {
```

Comment:
I didn't know about that header file but I don't it can be used here.

These are "runtime" tests which don't link against ``libHalide`` (which appears to contain ``Util.h``'s implementation).

What we kind of need here is a testing support library that is independent of ``libHalide``.